app/helpers/application_helper.rb

```diff
@@ -474,7 +474,7 @@ def user_set?(field)
   def notifications
     content_tag :div, id: 'toast-notifications-container',
                       'data-notifications': toast_notifications_data.to_json.html_safe do
-      mount_react_component('ToastNotifications', '#toast-notifications-container')
+      react_component('ToastNotifications')
```

Comment:
Can you also remove the `content_tag` from line 475?

Reply:
I can't because the notification load from `<div id='toast-notifications-container' data-notifications='some_notifications'>`